javaScriptLoops-forLoop/codeChallenge/randomColors.js

```diff
@@ -0,0 +1,21 @@
+let html = '';
+let red;
+let green;
+let blue;
+let randomRGB;
+
+const randomNumber = () => {
+  return Math.floor(Math.random() * 256);
+}
+
+for (let count = 1; count <= 10; count++) {
+  red = randomNumber();
+  green = randomNumber();
+  blue = randomNumber();
+
+  randomRGB = `rgb( ${red}, ${green}, ${blue} )`;
+
+  html += `<div style="background-color: ${randomRGB}">${count}</div>`;
+};
+
+document.querySelector('main').innerHTML = html;
```

Comment:
Great work, @nicoRDL! However, I'd like to challenge you to further enhance and optimize the code. Can you achieve the same result without using the variables `red`, `blue`, and `green`? Reducing the number of lines of code will lead to more concise and readable code. Here's a hint: I was able to accomplish this in just 10 lines of code. I used implicit return, and `randomRGB` was declared only once inside the loop.

Reply:
Challenge accepted @lycoangelo. Code updated here https://github.com/nicoRDL/dotTheory-JS/pull/12/commits I left the spaces and empty lines for readability sake, but the code does comply with the 10 lines of logic limit. 

For re-review and approval.

Reply:
Nice job, @nicoRDL! 🎉